Libplanet.Extensions.Cocona/Commands/StoreCommand.cs

```diff
@@ -95,6 +135,21 @@ private static BlockHash GetBlockHash(IStore store, int blockIndex)
             return blockHash;
         }
 
+        private static BlockHash GetBlockHash(IStore store, TxId txId)
+        {
+            if (!(store.GetFirstTxIdBlockHashIndex(txId) is { } blockHash))
+            {
+                throw Utils.Error($"cannot find the block with the TxId[{txId.ToString()}]");
+            }
+
+            if (store.IterateTxIdBlockHashIndex(txId).Count() > 1)
+            {
+                throw Utils.Error($"multiple BlockHashes found for TxId[{txId.ToString()}]");
```

Comment:
Instead of throw error, how about return all `BlockHash`es found? This may occur pretty often if reorg happens.

Reply:
I Introduced a new command `BlockHashesByTxId` instead of changing an existing one for the purpose.
Because `BlockByTxId` command prints the whole of the block itself, it would be messy when the command list up blocks it found. @limebell

Reply:
It could be useful to have a query for a block with the implemented behavior above. I'm not sure if it would be the least expected surprise though. Perhaps updating the command description may help. I think it might also be useful to query all the blocks with given `TxId`. How about having `GetBlocks()` to mirror the behaviour of `GetBlock()`?

Reply:
I make a compromise with you guys about the command.
It changed to `GetBlocks` and return a list of blocks found.
And `BlockHashesByTxId` still there to return a list of block hashes

@limebell @greymistcube